flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/operator/CepOperator.java

```diff
@@ -417,6 +405,9 @@ private void processEvent(NFAState nfaState, IN event, long timestamp) throws Ex
                             timestamp,
                             afterMatchSkipStrategy,
                             cepTimerService);
+            if (nfa.getWindowTime() > 0 && nfaState.isNewStartPartialMatch()) {
+                registerTimer(timestamp + nfa.getWindowTime());
```

Comment:
Should we add the following code in `onProcessingTime`? 
```
advanceTime(nfaState,timer.getTimestamp());
```

Otherwise, I'm afraid that the timer registered via `registerTimer(timestamp + nfa.getWindowTime())` will not trigger the nfa timed out in processing time.

Reply:
@dianfu thank a lot for the review . I found that I did miss this line. So I made the following changes:

1. I notice that the processing timer registered in the previous code is `timerService.currentProcessingTime() + 1`, so I modified it a bit.

2. As your suggestion add `advanceTime`  in `onProcessingTime`. But I prefer to use `timerService.currentProcessingTime()` 

And I have updated the PR ,PTAL